genome_designer/pipeline/read_alignment.py

```diff
@@ -142,9 +149,16 @@ def align_with_bwa_mem(alignment_group, sample_alignment):
                     stdout=fh, stderr=error_output,
                     shell=True, executable=settings.BASH_PATH)
 
+        # Set processing mask to not compute insert metrics if reads are
+        # not paired end, as the lumpy script only works on paired end reads
+        opt_processing_mask = {}
+        if not is_paired_end:
```

Comment:
Never mind. Looks like `process_sam_bam_file` fills in missing keys with values from `DEFAULT_PROCESSING_MASK`.